bindings/java/src/main/com/apple/foundationdb/Database.java

```diff
@@ -216,11 +218,13 @@ default CompletableFuture<Void> waitPurgeGranulesComplete(byte[] purgeKey) {
 	 *
 	 * @param purgeKey key to watch
 	 * @param e the {@link Executor} to use for asynchronous callbacks
+	 *
+	 * @return -- undocumented
```

Comment:
It returns void here. If it returns successfully, the data has been purged tracked by the provided purgeKey from purgeBlobGranules().